src/framework/global/thirdparty/kors_logger/src/logdefdest.cpp

```diff
@@ -110,12 +110,24 @@ void ConsoleLogDest::write(const LogMsg& logMsg)
     std::string msg;
     msg.reserve(100);
     msg.append(colorCode(logMsg.color));
-    msg.append(m_layout.output(logMsg));
+    std::string log = m_layout.output(logMsg);
+    msg.append(log);
     msg.append("\033[0m");
 #ifdef _WIN32
-    std::wstring temp = std::wstring(msg.begin(), msg.end());
-    OutputDebugString(temp.c_str());
+    std::wstring str = std::wstring(msg.begin(), msg.end());
+    size_t preview = 0;
+    for (size_t i = 0; i < str.size(); ++i) {
+        if (str.at(i) == L'\n') {
+            str[i] = L'\0';
+            OutputDebugString(&str[preview]);
```

Comment:
What will be the console output here? The code looks like it's one character, but I assume it should be a whole line

Reply:
The whole line will be.
OutputDebugString expected null terminated C string (`const wchar*`), 
so I  do null terminated - `str[i] = L'\0'`;
and put `const wchar*` - `&str[preview]`